### PR TITLE
Fix: Certificate email style

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

Certificate failure mails improvements:

- Email to be styled
- Email to be english-only

## Test Plan

- [x] Manual QA

After:

CleanShot 2024-01-20 at 13 49 07@2x

Before:

CleanShot 2024-01-20 at 12 31 51@2x

## Related PRs and Issues

x

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/config/locale/templates/email-certificate-failed.tpl [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+
+
+
+
+<p>Hello,</p>
+<p>Domain <strong>{{domain}}</strong> failed to generate certificate after <strong>{{attempts}}</strong> consecutive attempts with following error:</p>
```
Comment (christyjacob4) [inline app/config/locale/templates/email-certificate-failed.tpl:7]:
```suggestion

<p>Your domain <strong>{{domain}}</strong> failed to generate certificate after <strong>{{attempts}}</strong> consecutive attempts with the following error:</p>
```

File: app/config/locale/templates/email-certificate-failed.tpl [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+
+
+
+
```
Comment (christyjacob4) [inline app/config/locale/templates/email-certificate-failed.tpl:5]: why so many blank lines ?

File: app/config/locale/templates/email-certificate-failed.tpl [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    <tr>
+        <td align="center" style="border-radius: 8px; background-color: #ffffff;">
+            <p style="text-align: start; font-size: 14px; font-family: Inter; color: #414146; text-decoration: none; border-radius: 8px; padding: 32px; border: 1px solid #EDEDF0; display: inline-block; word-break: break-word;">{{error}}</p>
+        </td>
+    </tr>
+</table>
+
+<p>We suggest to follow the below steps:</p>
+<ol>
+    <li>Examine the logs above to identify the issue</li>
```
Comment (christyjacob4) [inline app/config/locale/templates/email-certificate-failed.tpl:19]:
```suggestion
    <li>Examine the logs above to try and identify the issue</li>
```

File: app/config/locale/templates/email-certificate-failed.tpl [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    <li>Ensure the domain did not expire without a renewal</li>
```
Comment (christyjacob4) [inline app/config/locale/templates/email-certificate-failed.tpl:20]:
```suggestion
    <li>Ensure your domain has not expired</li>
```

File: app/config/locale/templates/email-certificate-failed.tpl [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    <li>Check DNS configuration for any unwanted changes</li>
```
Comment (christyjacob4) [inline app/config/locale/templates/email-certificate-failed.tpl:21]:
```suggestion
    <li>Check your DNS configuration for any unexpected values</li>
```

Comment (christyjacob4) [reply app/config/locale/templates/email-certificate-failed.tpl:21]: You can also add another point to `Manually re-trigger a certificate generation from the Appwrite Console`

File: app/config/locale/templates/email-certificate-failed.tpl [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+</ol>
+
+<p>The previous certificate will be valid for 30 days since the first failure. We highly recommend investigating this issue, otherwise the domain will end up without a valid HTTPS communication.</p>
```
Comment (christyjacob4) [inline app/config/locale/templates/email-certificate-failed.tpl:24]:
```suggestion
<p>The existing certificate will remain valid for 30 days from the initial failure. It is highly recommended to investigate this issue; failing to do so will lead to security vulnerabilities.</p>
```

File: app/config/locale/translations/en.json [hunk 7]
```diff
@@ -30,12 +30,6 @@
     "emails.invitation.footer": "If you are not interested, you can ignore this message.",
     "emails.invitation.thanks": "Thanks",
     "emails.invitation.signature": "{{project}} team",
-    "emails.certificate.subject": "Certificate failure for %s",
-    "emails.certificate.hello": "Hello",
-    "emails.certificate.body": "Certificate for your domain '{{domain}}' could not be generated. This is attempt no. {{attempt}}, and the failure was caused by: {{error}}",
-    "emails.certificate.footer": "Your previous certificate will be valid for 30 days since the first failure. We highly recommend investigating this case, otherwise your domain will end up without a valid SSL communication.",
-    "emails.certificate.thanks": "Thanks",
-    "emails.certificate.signature": "{{project}} team",
```
Comment (christyjacob4) [inline app/config/locale/translations/en.json:38]: Where are we setting the Thanks part? I dont see it in the code below

Comment (christyjacob4) [reply app/config/locale/translations/en.json:38]: nor in the .tpl file above

Comment (Meldiron) [reply app/config/locale/translations/en.json:38]: We are removing it. It used to be set, but our new approach is to not have translation for developer-facing emails.
Ill remove it from all languages, not just EN.

File: app/config/locale/templates/email-certificate-failed.tpl (hunk 2 above)
Comment (Meldiron) [reply app/config/locale/templates/email-certificate-failed.tpl:5]: Removed.

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**